server/src/main/java/org/elasticsearch/index/mapper/IpFieldMapper.java

```diff
@@ -455,7 +462,75 @@ public BlockLoader blockLoader(BlockLoaderContext blContext) {
             if (hasDocValues()) {
                 return new BlockDocValuesReader.BytesRefsFromOrdsBlockLoader(name());
             }
-            return null;
+
+            if (isStored()) {
```

Comment:
I added this because it was very easy to do. I am not sure if there are any cases where this is not beneficial? Some existing implementations don't bother and just go straight to loading from source.

Reply:
👍 - seems easy to just do in this pr.